tools/stats/upload_external_contrib_stats.py

```diff
@@ -0,0 +1,142 @@
+import argparse
+import datetime
+import json
+import os
+import urllib.parse
+from typing import Any, Callable, cast, Dict, List, Optional, Set
+from urllib.error import HTTPError
+from urllib.request import Request, urlopen
+
+# import time
+from tools.stats.upload_stats_lib import upload_to_s3
+
+FILTER_OUT_USERS = set(["pytorchmergebot", "facebook-github-bot", "pytorch-bot[bot]"])
```

Comment:
`flake8-comprehensions` complains about unnecessary literal.

```suggestion
FILTER_OUT_USERS = {"pytorchmergebot", "facebook-github-bot", "pytorch-bot[bot]"}
```